### ISSUE TITLE
Prebuilt Dependency Warning with Webpack

### ISSUE THREAD

**Issue by joshhepworth** [braintree/braintree-web#189]
When importing braintree in Webpack, I get the following warning:

`This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.`

![screen shot 2016-08-09 at 4 11 28 pm](https://cloud.githubusercontent.com/assets/609235/17531925/13a66c6a-5e4c-11e6-984a-0a832d0bda3c.png)

There seems to be a [long running issue on webpack's repo](https://github.com/webpack/webpack/issues/1617) with lists of possible solutions – it seems that some of it could be related to building with browserify – if that's something you're doing.

To be clear, this does not affect the ability to run or deploy code (at least not obviously), but it is an annoyance when trying to filter for other actual issues while developing.

**Comment by mrak** [comment]
This appears to be a duplicate of https://github.com/braintree/braintree-web/issues/52. Yes, we do use browserify for development 😄  We do wish that webpack had a way to silence warnings such as these.

We don't have any plans to release unminified code of v2 our our library, aside from those files hosted on our domain.

v3, however, is open-source and we package unminified versions alongside the minified ones.

```javascript
// v3
var braintreeMinified = require('braintree-web');
var braintreeUnminified = require('braintree-web/debug');
```

There is a big __CAVEAT__ with using the debug (unminified) versions from v3 -- they will cause all iframe HTML, CSS, and JavaScript loaded by us to _also_ be unminified. It is intended for debugging purposes only.

**Comment by crookedneighbor** [comment]
We've released 3.8.0, which includes the src files on NPM which means you should no longer receive those warnings when compiling.
